qualang_tools/plot/README.md

```diff
@@ -55,38 +55,30 @@ In the example below you can see five parts:
 from fitting import *
 
 ### Simulating the measured data ###
-# x and y are the measured data
-k = 10
-kc = 5
-A = 10
-
-x = np.linspace(-100, 100, 200)
-y = (
-        kc * (k / (1 + (4 / k ** 2) * (x + 10) ** 2))
-        + 1 * (np.random.rand(len(x)) - 0.5)
-        + 200
-)
-
+# taus and amplitude are the measured data
+def ramsey_data(x, final_offset, T2, amp, initial_offset, f, phase):
+    return  final_offset * (1 - np.exp(-x * (1/T2))) + amp / 2 * (
+            np.exp(-x * (1/T2))
+            * (initial_offset * 2 + np.cos(2 * np.pi * f * x + phase))
+            )
+
+
+tau_min = 4
+tau_max = 100
+dtau = 2
+taus = np.arange(tau_min, tau_max + 0.1, dtau)
+amplitude = ramsey_data(8 * taus, final_offset=0.1, T2=300, amp=1, initial_offset=0, f=10e-3, phase=0.01) + np.random.normal(

```

Comment:
Yes I can explain :)
Since I took the data in QUA, taus was in clock cycles so I multiply it by 4 and for the last plot it was actually an echo so I added this additional factor of 2 (8 in total) to convert taus in ns.
Here I am assuming that tau is already in ns so no need for this extra conversion

Reply:
because in the way we define it you play pi/2 - tau - pi - tau - pi/2 so the total time is 2*tau